packages/docusaurus-plugin-stored-data/src/index.ts

```diff
@@ -78,6 +83,16 @@ const getData = async (location: string): Promise<string> => {
   throw new Error(`Could not determine if ${location} is a URL or file`);
 };
 
+
+const  validatingURLusingRegex = (locationString : string): Boolean => {
```

Comment:
Since this is only checking if the protocol is valid, not the rest of the URL, I would recommend either:

- Renaming the function to something like `isHttpOrHttps(locationString: string): boolean`
- Validating the rest of the URL with something like the following (untested)

```ts
const validateUrl = (location: string): boolean => {
  try {
    const url = new URL(location);
    return url.protocol === "http:" || url.protocol === "https:";
  } catch {
    return false;
  }
};
```

Reply:
Okay, Thank you @mrjones2014 :) for the feedback. I am new to Open Source so just trying my best to contribute. I will update it and raise the PR again :)

Reply:
Love to see new open source contributors ❤️ 

Please feel free to let me know if you need any help!

Reply:
Hi @mrjones2014 ,
Updated the branch and implemented the second option you suggested. Please review it :)
Thank you